docker/install-scripts/ansible.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+set -e
+
+while [ $# -gt 0 ]; do
```

Comment:
It is better to just take a arg list rather than option flags to avoid this.

Reply:
won't fix